backend/app/tests/api/routes/test_responses.py

```
@@ -123,3 +123,150 @@
+    assert conversation_data.input_tokens == 10
+    assert conversation_data.output_tokens == 5
+    assert conversation_data.total_tokens == 15
+    assert conversation_data.assistant_id == "assistant_123"
+    assert conversation_data.project_id == glific_project.id
+
+
+@patch("app.api.routes.responses.OpenAI")
+@patch("app.api.routes.responses.get_provider_credential")
+@patch("app.api.routes.responses.get_assistant_by_id")
+def test_responses_sync_endpoint_stores_conversation(
+    mock_get_assistant,
```

Ruff (0.12.2)

Unused function argument: `mock_get_assistant`

(ARG001)